install-deps.sh

```diff
@@ -65,8 +65,8 @@ function munge_debian_control {
 	sed -i -e 's/^# Crimson[[:space:]]//g' $control
     fi
     if $with_jaeger; then
-	sed -i -e 's/^# Jaeger[[:space:]]//g' $control
-	sed -i -e 's/^# Crimson      libyaml-cpp-dev,/d' $control
+	sed -i -e 's/^# Jaeger[[:space:]]\+//g' $control
```

Comment:
```suggestion
      sed -i -e 's/^# Jaeger[[:space:]]\+/               /g' $control
      sed -i -e '/^# Crimson[[:space:]]\+libyaml-cpp-dev,/d' $control
      sed -i -e 's/^[[:space:]]\+Built-Using:/Built-Using:/' $control
```
we need some indentation otherwise control file has parsing error, check make check logs for more details

Reply:
@ideepika I don't see any error for parsing error in make check logs. Could point to them, please?

Reply:
aah, sorry I thought the parsing failure is due to debian/control. 
But I tested your changes locally, which was my point of reference 
```
mk-build-deps: error: syntax error in control file `debian/control' at line 76: line with unknown format (not field-colon-value)
```
line 76: 
```
     libyaml-cpp-dev,
               lsb-release,
nlohmann-json-dev | nlohmann-json3-dev,
               parted,
```

Reply:
@ideepika You are right. Please review both this PR and #38744
With both of them, jaeger with built correctly!

Reply:
@clwluvw we specifically require libyaml-dev +0.6.0 version which is without boost as a dependency, otherwise it will conflict with cephboost version and hence will break dependency build, the above suggestion would take care of that the yaml version is always 0.6.0 or above